things/api/things/http/responses.go

```diff
@@ -63,6 +63,33 @@ func (res thingRes) Empty() bool {
 	return true
 }
 
+type thingsRes struct {
+	Things  []thingRes `json:"things"`
+	created bool
+}
+
+func (res thingsRes) Code() int {
+	if res.created {
+		return http.StatusCreated
+	}
+
+	return http.StatusOK
+}
+
+func (res thingsRes) Headers() map[string]string {
+	if res.created {
+		return map[string]string{
+			"Location": "/things/provision",
```

Comment:
I would use something else than `/things/provision` to set the Location.

Reply:
The other responses that return a list of values have an empty location.  I'll go ahead and change this to match.